picard/script.py

```diff
@@ -1165,7 +1182,8 @@ def func_is_complete(parser):
 @script_function(documentation=N_(
     """`$firstalphachar(text,nonalpha="#")`
 
-Returns the first character of `text`. If `text` is not an alphabetic character `nonalpha` is returned instead.
+Returns the first character of `text`.
+If `text` is not an alphabetic character `nonalpha` is returned instead, if not passed `#`.
```

Comment:
I find the part ", if not passed #" hard to understand. What about "..., which defaults to #" or maybe more explicit "If nonalpha is not passed, # is used."?

Reply:
I addressed this and other issues, rebased.